encrypted-media-respec.html

```diff
@@ -606,7 +606,83 @@
           <section>
             <h5>Get Supported Configuration</h5>
             <p>Given a <a def-id="keysystems"></a> implementation <var>implementation</var>, <a>MediaKeySystemConfiguration</a> <var>candidate configuration</var>, and <var>origin</var>, this algorithm returns a supported configuration or <code>null</code> as appropriate.</p>
-            <p class="note">Unrecognized dictionary members in <var>candidate configuration</var> are ignored per [[WebIDL]] and will never reach this algorithm. Thus, they cannot be considered as part of the configuration.</p>
+            <p class="note">Unrecognized dictionary members in <var>candidate configuration</var> are ignored per [[WebIDL]] and will never reach this algorithm. Thus, they cannot be considered as part of the configuration.
+            </p>
+            <ol>
+              <li>
+                <dl class="switch">
+                  <dt>
+                    If the <a def-id="option-distinctiveIdentifier"></a> member of
```

Comment:
I think we might also end up with more common steps here. For example, "If _supported configuration_ is `ConsentDenied`, &lt;run the Get Supported Configuration and Consent algorithm with the opposite value&gt;."